src/peripheral/rcc/clock_control.rs

```diff
@@ -174,3 +207,208 @@ impl CR2 {
         (self.0 & mask) != 0
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_cr_set_clock_on() {
```

Comment:
When a specific clock is being tested, should put the clock name the function name. There are other functions below that this applies to as well.